scripts/release.js

```diff
@@ -0,0 +1,48 @@
+const shell = require('shelljs');
+const semver = require('semver');
+const conventionalRecommendedBump = require('conventional-recommended-bump');
+
+async function getNextVersion() {
+  // get version from root package.json
+  // need to remove double quotes ("") from version string
+  const VERSION = shell.exec('npm pkg get version').trim().replace(/"/g, '');
```

Comment:
Can't we get the version simply by doing it like this?
https://github.com/inovex/elements/blob/bfdfc0205fa3aed18f8667ca9f8cef3d5c10b834/scripts/bumprootversion.js#L3-L6